### PR TITLE
User Pojo Created.!

### PR DESCRIPTION
**Pull request by samalasaibaba**
Pojo created for User

### CODE REVIEW

File: src/main/java/com/pojos/User.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.pojos;
+
+public class User {
+	private Integer userId;
+	private String userName;
+	private String emailId;
+	private String mobileNumber;
+	private String Address;
+
```
Comment (samalasaibaba1993) [inline src/main/java/com/pojos/User.java:9]: create setter and getters

Comment (samalasaibaba) [inline src/main/java/com/pojos/User.java:9]: add setters and getters

Comment (samalasaibaba1993) [reply src/main/java/com/pojos/User.java:9]: Ok

**Review by samalasaibaba — COMMENTED** [review]
Done

File: src/main/java/com/pojos/User.java (hunk 1 above)
Comment (samalasaibaba) [reply src/main/java/com/pojos/User.java:9]: Ok I'll

File: src/main/java/com/pojos/User.java [hunk 2]
```diff
@@ -6,5 +6,37 @@
 	private String emailId;
 	private String mobileNumber;
```
Comment (samalasaibaba1993) [inline src/main/java/com/pojos/User.java]: It is better to change String to Integer for mobileNumber

Comment (samalasaibaba) [reply src/main/java/com/pojos/User.java]: Ok I'll make changes as for your requirement

**Comment by samalasaibaba1993** [comment]
You changed data type but where is setters and getters for mobile number

**Pull request merged**